### ISSUE TITLE
distance_scale 25

### ISSUE THREAD

**Issue by w-hc** [apchenstu/TensoRF#14]
Hi thanks a lot for releasing the code for the nice work

`distance_scale=25` is used at train and test time for rendering, but not for alpha mask update or mesh extraction. I can't seem to find discussion regarding this hyperparameter in the main text, and I wonder whether you could provide a little more explanations. 

It appears that using `distance_scale` for mesh extraction results in noisier geometry with more floaters, while not applying `distance_scale` during training would lead to failed optimization. Thanks again.

**Comment by apchenstu** [comment]
Hi, thanks! It's equivalent to mod with distance_scale and adjusts the thresholds in building the binary volume or marching cude

**Comment by gaoxiangjun** [comment]
Hi, could you explain that why not applying distance_scale during training would lead to divergence ?

**Comment by gaoxiangjun** [comment]
BTW, could you please explain what does this code block mean ? 
`rate_a = (self.aabb[1] - rays_o) / vec
        rate_b = (self.aabb[0] - rays_o) / vec
        t_min = torch.minimum(rate_a, rate_b).amax(-1).clamp(min=near, max=far)`

**Comment by Xiaoming-Zhao** [comment]
Hi @apchenstu, I tried to disable `distance_scale` and `density_shift` during training. Namely, I set `distance_scale = 1` and `density_shift = 0`. After 30k-iter training, this gives me test PSNR of 33.93 on the Lego scene, which is quite inferior to the 36.46 reported in the paper.

May I know whether you mind sharing some intuition why we need such a scale and shift during training to boost the performance? Thanks a lot.
